launcher/ui/dialogs/BlockedModsDialog.cpp

```diff
@@ -1,14 +1,41 @@
+// SPDX-FileCopyrightText: 2022 Sefa Eyeoglu <contact@scrumplex.net>
+// SPDX-FileCopyrightText: 2022 Rachel Powers <508861+Ryex@users.noreply.github.com>
```

Comment:
Is it useful to have emails in copyright headers if it's a noreply?

Reply:
i think we should just not add the email in those cases

Reply:
while it may not directly serve as a contact point it does effectively serve as identifying the github account attached to the name. that said. I'm not hurt if my name is left out of the license text.

Reply:
agree with ryex, it does help with identifying the gh account but isnt really necessary, and i am also fine with being left out of the license text